src/appdistribution/client.ts

```diff
@@ -174,4 +178,44 @@ export class AppDistributionClient {
 
     utils.logSuccess("distributed to testers/groups successfully");
   }
+
+  async addTesters(projectName: string, emails: string[]) {
+    const appDistroV2Client = new Client({
+      urlPrefix: api.appDistributionOrigin,
+      apiVersion: "v1",
+      auth: true,
+    });
+    try {
+      await appDistroV2Client.request({
+        method: "POST",
+        path: `${projectName}/testers:batchAdd`,
+        headers: { "X-Client-Version": `${pkg.name}/${pkg.version}` },
+        body: { emails: emails },
+      });
```

Comment:
If you prefer, I _think_ that this can be simplified (and typed) a bit:

```typescript
await appDistroV2Client.post<BatchRemoveTestersResponse, BatchRemoveTestersResponse>(
  `${projectName}/testers:batchAdd`,
  { emails: emails },
  { headers: { "X-Client-Version": `${pkg.name}/${pkg.version}` } });
```

Reply:
I was having a hard time getting the right response value from it afterwards so I think that the way we have it currently is fine.

Reply:
Let me know what the difficulty was at some point (maybe message me directly) and let's see if we can't improve it some way. :)